include/sharg/detail/format_base.hpp

```diff
@@ -321,14 +325,26 @@ class format_help_base : public format_base
         for (auto f : positional_option_calls)
             f();
 
-        // add options and flags if specified
-        if (!parser_set_up_calls.empty())
-            derived_t().print_section("Options");
+        // There are always options because of the common options
+        derived_t().print_section("Options");
 
         // each call will evaluate the function derived_t().print_list_item()
         for (auto f : parser_set_up_calls)
             f();
 
+        // print Common options after developer options
+        derived_t().print_subsection("Common options:");
```

Comment:
Do we want to print `Common options:` or `Common options`?
I know we did `Basic options:` before, but without `:` it kinda looks better?